app/models/user/permissions.rb

```diff
@@ -56,6 +53,26 @@ def member_permission_ids=(roles)
     self.admin_sections = (Array(roles).reject(&:blank?).map(&:to_sym) & AdminSection.permissions)
   end
 
+  
+  # TODO: We can't use `alias_attribute` anymore because its deprecated for this scenario (not a real attribute),
+  # and `alias_method` doesn't work either, throwing "unknown attribute 'allowed_sections' for User."
```

Comment:
How did you `alias_method`? Did you alias `allowed_sections=` in addition to `allowed_sections`?

Reply:
Hm... good question. I don't remember - could be that I only aliased `allowed_sections`, and not `allowed_sections=`. I'm not sure it would make a difference though, but I could try.

Reply:
It should make a difference. `#something` and `#something=` are unrelated as far as ruby is concerned.

Reply:
Yeah, what I mean is that I am not sure having `#something` and `#something=` is enough for `something` to be considered an attribute. But I'll try that!

Reply:
Yeah, makes sense